### PR TITLE
chore: add launch config for VSCode

### PR DESCRIPTION
**Pull request by jonas-jonas**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  // Launch configuration to build and launch Kratos
```
Comment (aeneasr) [inline .vscode/launch.json:2]: does this parse?

Comment (jonas-jonas) [reply .vscode/launch.json:2]: yep, VS Code launch configs allow comments.

File: Makefile [hunk 2]
```diff
@@ -16,7 +16,8 @@ GO_DEPENDENCIES = github.com/ory/go-acc \
 				  github.com/go-swagger/go-swagger/cmd/swagger \
 				  golang.org/x/tools/cmd/goimports \
 				  github.com/mattn/goveralls \
-				  github.com/cortesi/modd/cmd/modd
+				  github.com/cortesi/modd/cmd/modd \
+				  github.com/mailhog/MailHog
```
Comment (alnr) [inline Makefile:20]: Maybe a good idea to add this to `go_mod_indirect_pins.go`.

**Review by alnr — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (jonas-jonas) [reply Makefile:20]: Can do, could you explain, what that the point of that file is? I don't quite understand it.

Comment (alnr) [reply Makefile:20]: This pattern is used to manage/pin versions of tools the repo uses (like code generators, coverage calculators, testing utilities like MailHog) using the `go.mod` file of the actual Go module -- kratos in this case.

`go_mod_indirect_pins.go` is excluded from compilation via build tags which are never set. In fact, that file cannot be compiled at all. But it is part of the dependency resolution, and it's dependencies are noted down in `go.mod`. It serves only to import (as `_`) all the tools required in that repo, which are themselves written in Go.

Comment (jonas-jonas) [reply Makefile:20]: ah, I see. Thanks, I'll add Mailhog there as well.

**Review by alnr — APPROVED** [review]

**Pull request merged**